woocommerce-gateway-stripe.php

```diff
@@ -207,6 +208,13 @@ public function init() {
 						require_once dirname( __FILE__ ) . '/includes/admin/class-wc-stripe-onboarding-controller.php';
 						new WC_Stripe_Onboarding_Controller();
 					}
+
+					// If UPE is enabled and there are enabled payment methods, we need to load the disable Stripe confirmation modal.
+					$enabled_upe_payment_methods = get_option( 'woocommerce_stripe_settings', [] )['upe_checkout_experience_accepted_payments'];
+					$upe_payment_requests_enabled = 'yes' === get_option( 'woocommerce_stripe_settings', [] )['payment_request'];
```

Comment:
Moving these checks (with the exclusion of `WC_Stripe_Feature_Flags::is_upe_checkout_enabled()`, which seems to be applied in these files) to the `includes/admin/class-wc-stripe-admin.php` file would help colocate the logic, what do you think?

It could be placed in the constructor (before registering the hooks) or on each hook as well 🤷

Reply:
Yep...that makes sense. I had them in the other file originally, but then moved them here before making the PR. It makes sense, though, to split them out to checking if UPE is enabled in this file, THEN check in the included file if the other conditions are met to need the modal.